modules/ravestate_ui_new/README.md

```diff
@@ -0,0 +1,21 @@
+# Ravestate UI
+
+Roboy ravestate UI module. 
+
+## Setup, Development and Building
+
+This project is built with Angular 8 and TypeScript. 
+
+Initial setup:
+- install Node.js and npm
+- Install project dependencies by running `npm install` in the root folder (= folder with `package.json`) 
+ 
+To start development: 
+- run `ng serve` in the root folder to start a dev server
```

Comment:
For me (on Ubuntu 16.04) `ng serve` did not work, but `npm start` did the trick. Is this a common problem that should be mentioned in the README?

Reply:
Good catch! You need to install the Angular CLI globally by running `npm install -g @angular/cli`. 
The readme was updated. Please give it a try

Reply:
I realized that my .npm-global was not in the path, adding that fixed the issue :goat: